engine_graphics/BUILD

```diff
@@ -3,7 +3,7 @@ cc_library(
     srcs = glob([
         "src/**/*.cpp",
     ],
-    exclude = ["src/core_test_node.cpp"],
+    exclude = ["src/core_test_node.cpp", "src/renderer_test_node.cpp", "src/main.cpp"],
```

Comment:
Cleanup unneeded sources, and executables.

Reply:
Done